comp230_01_1607539.tex

```diff
@@ -0,0 +1,65 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage{url}
+\usepackage[autostyle]{csquotes}
+
+% Please include a clear, concise, and descriptive title
+\title{What are the risks associated with nurturing a modding community, and how can game developers protect themselves?}
+
+% Please do not change the subtitle
+\subtitle{COMP230}
+
+% Please put your student number in the author field
+\author{Student number: 1607539}
+
+\begin{document}
+
+\maketitle
+
+Game developers can encourage modifications by releasing the game's source code to the public. An often repeated example is the case of \textit{Doom} (1993) in which the media files were separated from the main program and made accessible for users \cite{sotamaa2010game}. This is historically celebrated as a consequence of mutual respect \cite{hong2013game}. Modders use these toolsets to build mods of a number of different scales, from the massive ``total conversions'' accomplished by teams of modders who remake entire games with new artwork, storylines, and gameplay to smaller mods such as ``skin'' (aesthetic tweaks to specific graphic models) \cite{hong2014becoming}.  Generally going far beyond what is designed by the commercial developers \cite{postigo2007mods}.
```

Comment:
You could introduce the term `"mods"` in this opening paragraph. You have used the term in several places.

Reply:
`Generally going far beyond what is designed by the commercial developers \cite{postigo2007mods}.` This sentence seems stuck-on and doesn't flow with the rest of the paragraph. I am wondering if the point Postigo is actually making  could be better worked into the next paragraph as part of the challenge that motivates your work.